File: java/latihanempat/src/main/java/com/android/cnd/latihanempat/BiodataActivity.java
package com.android.cnd.latihanempat;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class BiodataActivity extends AppCompatActivity {
    TextView name;
    String namaMhs = "Maya";

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_biodata);
        name = findViewById(R.id.your_name);

        name.setText(namaMhs);
    }
}

File: kotlin/latihanenam/src/main/java/com/android/cnd/latihanenam/BiodataActivity.kt
package com.android.cnd.latihanenam

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class BiodataActivity : AppCompatActivity() {
    private lateinit var name: TextView
    private var namaMhs : String? = "Maya"

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_biodata)
        name = findViewById(R.id.your_name)
        name.text = namaMhs
    }
}